webapp-django/crashstats/manage/admin.py

```diff
@@ -0,0 +1,199 @@
+import hashlib
+
+import requests
+from six.moves.urllib.parse import urlparse
+
+from django import http
+from django.conf import settings
+from django.contrib import messages
+from django.core.cache import cache
+from django.db import connection
+from django.shortcuts import redirect, render
+
+from crashstats.manage.decorators import superuser_required
+from crashstats.manage import forms
+from crashstats.manage import utils
+from crashstats.supersearch.models import SuperSearchMissingFields
+from crashstats.crashstats.models import GraphicsDevices
+from crashstats.crashstats.utils import json_view
+
+
+@superuser_required
+def crash_me_now(request):
+    # NOTE(willkg): This intentionally throws an error so that we can test
+    # unhandled error handling.
+    1 / 0  # noqa
+
+
+@superuser_required
```

Comment:
Most of the rest of this admin view code is just copied over from the old admin views.